fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java

```diff
@@ -62,7 +59,7 @@ public Boolean hasReturn() {
      * @return true if the header has a return tag
      */
     public Boolean hasHandling() {
-        return any(preferTags(), getPreferTag("handling"));
+        return preferTags().stream().anyMatch(x -> x.getTag().equals("handling"));
```

Comment:
Oh, okay. I read that now. You'd theoretically need to sprout and zip, but that's not what Java Streams are for.

---
A. Soroka
The University of Virginia Library

On Jun 2, 2015, at 2:20 PM, Aaron Coburn <notifications@github.com> wrote:

> In fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/domain/SinglePrefer.java:
> 
> > @@ -62,7 +59,7 @@ public Boolean hasReturn() {
> >       * @return true if the header has a return tag
> >       */
> >      public Boolean hasHandling() {
> > -        return any(preferTags(), getPreferTag("handling"));
> > +        return preferTags().stream().anyMatch(x -> x.getTag().equals("handling"));
> 
> No dice on that -- the function returns a PreferTag and needs to filter on a String, so we can't map to a string and expect to get a PreferTag back.
> �
> �
> Reply to this email directly or view it on GitHub.
>